staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -587,19 +587,32 @@ func (r *Request) tryThrottle(ctx context.Context) error {
 	err := r.rateLimiter.Wait(ctx)
 
 	latency := time.Since(now)
+
+	var message string
+	switch {
+	case len(retryInfo) > 0:
+		message = fmt.Sprintf("Waited for %v, request: %s:%s - %s", latency, r.verb, r.URL().String(), retryInfo)
+	default:
+		message = fmt.Sprintf("Waited for %v due to client-side throttling, not priority and fairness, request: %s:%s", latency, r.verb, r.URL().String())
```

Comment:
I think `Waited for %v due to client-side throttling, before the first round trip` is better here.

Reply:
I think the meaning to the potential reader will be the same either way.